tool/src/java/org/sakaiproject/gradebookng/tool/panels/AddGradeItemPanelContent.html

```diff
@@ -1,15 +1,15 @@
-<!DOCTYPE html>
-<html xmlns:wicket="http://wicket.apache.org/dtds.data/wicket-xhtml1.4-strict.dtd">
+<!--<!DOCTYPE html>-->
+<!--<html xmlns:wicket="http://wicket.apache.org/dtds.data/wicket-xhtml1.4-strict.dtd">-->
 
-    <body>
-        <wicket:panel>
+    <!--<body>-->
+        <wicket:panel xmlns:wicket="http://wicket.apache.org">
```

Comment:
I added it there since I commented out the html tag which had one.  But, these changes here at the top can probebly be removed.  That was when I was trying to figure out why my form submit wasn't working.